agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -478,6 +480,10 @@ bool slave_thread::handle_cmdu_control_ieee1905_1_message(Socket *sd,
         return handle_channel_preference_query(sd, cmdu_rx);
     case ieee1905_1::eMessageType::CHANNEL_SELECTION_REQUEST_MESSAGE:
         return handle_channel_selection_request(sd, cmdu_rx);
+    case ieee1905_1::eMessageType::CLIENT_STEERING_REQUEST_MESSAGE:
+        return handle_client_steering_request(sd, cmdu_rx);
+    case ieee1905_1::eMessageType::ACK_MESSAGE:
```

Comment:
I may get in conflict with @morantr here :-) but I prefer a separate commit for the handling of the ACK message.

However, at least on the longer term, I think it's more appropriate if the ACK messages are handled in the transport. I believe that retransmissions are currently done there as well.

Reply:
Where did you get this idea? I have the same opinion as you. :)

Reply:
I'll update my changes soon.